src/main/java/org/jabref/logic/importer/fetcher/WorldcatFetcher.java

```diff
@@ -0,0 +1,87 @@
+package org.jabref.logic.importer.fetcher;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.jabref.logic.importer.EntryBasedFetcher;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.ParserResult;
+import org.jabref.logic.importer.fileformat.WorldcatImporter;
+import org.jabref.logic.net.URLDownload;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+/**
+ * EntryBasedFetcher that searches the Worldcat database
+ * 
+ * @see https://www.oclc.org/developer/develop/web-services/worldcat-search-api/bibliographic-resource.en.html
+ */
+public class WorldcatFetcher implements EntryBasedFetcher {
+
+	private final static String NAME = "Worldcat Fetcher";
+	public final static String API_KEY = "API-KEY-GOES-HERE";
+
+	private final static String WORLDCAT_OPEN_SEARCH_URL = "http://www.worldcat.org/webservices/catalog/search/opensearch?wskey=" + API_KEY;
+
+	@Override
+	public String getName() {
+		return NAME;
+	}
+
+	/**
+	 * Create a open search query with specified title
+	 * @param title the title to include in the query
+	 * @return the earch query for the api
+	 */
+	private String getOpenSearchURL(String title){
+		String query = "&q=srw.ti+all+\"" + title.replaceAll(" ", "%20") + "\"";
```

Comment:
Instead of manually replaying the illegal chars, I would create a URL (java.net.URL) and therefore let the java system handle the encoding of forbidden chars.